customising/cobrand-module.md

```diff
@@ -70,3 +70,13 @@ have a body that only covered the state council administrative area.
     This is used to turn the full authority name returned by MapIt into a short
 name.
 
+* send_questionnaires
+
+    By default, FixMyStreet [sends questionnaires]({{site.baseurl}}running/surveys)
+    (sometimes called _surveys_) four weeks after a user submitted a report. The
```

Comment:
Should we be consistent in naming? The front end consistently calls them questionnaires, apart from the one admin page that calls them surveys. The documentation seems to prefer surveys; was there a reason for this?